docs/index.md

```diff
@@ -25,7 +25,7 @@ From within an empty folder, add a minimal HTML file called `index.html`:
     ContainerJS status: <span id="status">initialising</span>
   </div>
 
-  <script src="https://unpkg.com/containerjs-api-bundle@0.0.2/build/containerjs-bundle.js"></script>
+  <script src="/scripts/containerjs-api.js"></script>
```

Comment:
I'd prefer to leave this as an unpkg.com link - this example is for first-time users, so should be as simple as possible!